src/Yoti.Auth/YotiClientEngine.cs

```diff
@@ -244,14 +256,25 @@ private static void HandleOtherAttributes(YotiUserProfile profile, AttrpubapiV1.
             }
         }
 
-        private string GetEndpoint(string token, string nonce, string timestamp, string sdkId)
+        private string GetEndpoint(HttpMethod httpMethod, string path, string token, string nonce, string timestamp, string sdkId, byte[] payload)
         {
-            return string.Format("/profile/{0}?nonce={1}&timestamp={2}&appId={3}", token, nonce, timestamp, sdkId);
+            string endpoint = string.Format("/{0}/{1}?nonce={2}&timestamp={3}&appId={4}", path, token, nonce, timestamp, sdkId);
+            if (payload != null && payload.Length > 0)
+            {
+                string urlSafePayloadString = Conversion.BytesToUrlSafeBase64(payload);
```

Comment:
It should be base64encoded, not base64URLencoded

Reply:
So will the URL be handled correctly if the payload contains "`+`" or "`/`"s?

Reply:
After seeing your below comment - this makes sense. So it's not being added to the URL, and so doesn't need to be URL safe